### ISSUE TITLE
Shortcuts not working 

### ISSUE THREAD

**Issue by tiefi1908** [Kemmey/Kemmey-TeslaWatch-Public#506]
I be bought yesterday the app. Ikbits working on iPhone . When I want to make a Shortcut on my Apple Watch series 5 for answer get  : 
It’s happens error. Wie can’t close the Action. WFBackroundShirtcutRunnerErrorDomain-Fehler 1 is comeing. this ist translated from German
I think I’m not alone here and we need help immediately. I have opened on watch the point Siri work with watch for Tesla. In the shortcuts, I activate it to play on Apple Watch. I have unpaired my watch But nothing helps
![image](https://user-images.githubusercontent.com/114850396/193442840-fde49b56-6491-4e1c-acfd-5e280c3edd42.jpg)
![image](https://user-images.githubusercontent.com/114850396/193442870-dbe97604-3ab3-4332-9df0-7ac8e08c935d.jpg)

**Comment by Kemmey** [comment]
Hmmm - that's a new one - I've never seen that before. 

Can you successfully run the shortcut on your phone from within the shortcuts app?

Can you successfully run the shortcut on your watch from within the shortcuts app? 

Which watchOS and iOS versions are you running?

What build number of Watch for Tesla are you running? (Visible at the top of the settings screen)

**Comment by Kemmey** [comment]
I'm in contact just now with another customer with same issue - he was not logged in on the watch app - could you verify you're logged in on the watch and can successfully execute commands from within the actual app?

Cheers,

Kim

**Comment by tiefi1908** [comment]
Hello. With ios9 16.2 now it’s working Thanks a lot Expon-SolutionBayerwaldstr. 38D-94161 Ruderting www.expon-solution.deAm 04.10.2022 um 20:09 schrieb Kim Hansen ***@***.***>:﻿
I'm in contact just now with another customer with same issue - he was not logged in on the watch app - could you verify you're logged in on the watch and can successfully execute commands from within the actual app?
Cheers,
Kim

**Comment by vtran54** [comment]
Hello, new user here. I'm wondering if I am seeing the same issue. I've added the shortcuts to my apple watch via my phone app, but the shortcuts are not being reflected in the Shortcuts app on my apple watch (so I'm unable to use Siri commands via my watch). 
The actions work from the Watch for Tesla app on the phone and on the watch; it's just the shortcuts that are not syncing.

Apple watch series 6 (wOS 9.1)
iPhone 12 (iOS 16.1.1)

Thanks!

**Comment by Kemmey** [comment]
Hey @vtran54 

Try restarting both phone and watch - seems like the watch connectivity framework has stalled. Or iCloud sync is stalled. Either way, it's entirely in apples systems the problem lies - nothing I can do unfortunately :-( 

Fingers crossed restarting helps...

**Comment by vtran54** [comment]
> 

Hi @Kemmey, quick update: restarting both devices did not do the trick. I've also unpaired the watch then re-paired, but still no watch shortcuts. 😕

Unfortunately this was the feature I was most excited about. Just curious, has this issue been reported by others as well? If this is an isolated incident I'm wondering if I need to open a ticket with Apple.

**Comment by Kemmey** [comment]
A few incidents - most were users not manually setting the shortcuts to sync from the phone (why on earth apple broke this is beyond me - do they not want people to use Siri on the watch?!) - others had deleted the shortcuts app off the watch. Most are fixed by restarting, as it's usually watchconnectivity or iCloud sync that's broken. Usual apple software quality these days...

Definitely open a ticket with apple on this one. I mean - make any simple shortcut that is not related to my app and get it to sync.

**Comment by Kemmey** [comment]
Btw - Siri and shortcuts are MUCH more reliable with the latest beta release with cloud processing enabled, as execution is handed off to the cloud and you're no longer limited to the 7 seconds of background processing allowed with watchOS 9!

Grab it here if you want to test it out:

https://testflight.apple.com/join/FdzFOLZv

**Comment by vtran54** [comment]
@Kemmey 
Good news! The shortcuts are now showing up 🙂. Though it may have been the result of two things:
1. I downloaded the beta version as you mentioned.
2. I noticed my cloud storage was full, so I expanded the storage.

Either way, it worked! Thanks for the quick replies

![165C6CFE-B450-47A7-9FD9-599EAF9C0379](https://user-images.githubusercontent.com/119393831/204629052-4108c286-c601-4ec9-b7fc-de31ba0140be.png)

**Comment by Kemmey** [comment]
Aaaah! Noted for future debugging - cloud storage being full would block iCloud syncing of anything, so definitely the culprit!

**Comment by Kemmey** [comment]
Remember to enable cloud processing from advanced settings, if you want to test out the enhanced Siri support. And do read the disclaimer - this feature means your access token (which is valid for up to 8 hours) will leave your device (fully encrypted of course) and be sent to an azure function which will keep your token in memory for up to 1 minute and 15 seconds.

**Comment by AxelFooley** [comment]
Sorry to resume this thread, i am having the same problem. I can run the shortcut manually from both the phone and the watch, i can run the shortcut through siri voice command on the phone, but on the watch Siri just makes a google search.

I am running the latest beta, i've activated the cloud processing, and i've enabled Watch for tesla to be managed through Siri in the watch and phone settings.

Any help will be appreciated.

**Comment by Kemmey** [comment]
@AxelFooley hah - yeah - that's Siri for ya! I have a Siri shortcut named "Heat the car" and Siri will consistently tell me that it is not able to do that. Because the word "Car" triggers Siris built-in Car-related features, and so it never goes to check that there's a shortcut named EXACTLY the phrase I uttered, and that it understood. The internal parsing of Siri just doesn't branch to the interpretation that I intended. 

Long story short - try renaming your shortcuts on your phone, and use a different trigger phrase. And avoid the word "Car"!

**Comment by haritrivedi** [comment]
same issues here - has anyone figured it out?
